core/server_config.go

```diff
@@ -56,6 +58,7 @@ type ServerConfig struct {
 	Strictmode          bool              `koanf:"strictmode"`
 	InternalRateLimiter bool              `koanf:"internalratelimiter"`
 	Datadir             string            `koanf:"datadir"`
+	PKI		    pkiconfig.Config	   `kaonf:"pki"`
```

Comment:
typo in koanf, so untested? also weird indentation

Reply:
It's still untested, but the typo is fixed. With the upcoming engine changes and default values I don't know how we want to test this yet.